### PR TITLE
Start conversation on request details page

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

File: src/views/Request.svelte [hunk 1]
```diff
@@ -24,9 +25,9 @@ $: provider = request.provider || {}
 $: conversations = request.threads || []
 $: isMine = me.id && (requestor.id === me.id)
 $: imProviding = me.id && (provider.id === me.id)
-$: hasConversation = conversations.length > 0
+$: hasConversation = conversations.length > 0 || potentialConversation
 $: destination = request.destination && request.destination.description || ''
-$: if (!conversationId && conversations.length > 0) {
+$: if (!potentialConversation && !conversationId && conversations.length > 0) {
```
Comment (wcjr) [inline src/views/Request.svelte]: assigning a `default` more declaratively would be helpful.

File: src/views/Request.svelte [hunk 2]
```diff
@@ -47,6 +48,25 @@ function asReadableDate(timestamp) {
...
 }
+
+function discussThis() {
+  potentialConversation = {
+    post: request
+  }
+}
+
+function onConversationStarted(event) {
+  let newConversation = {
```
Comment (wcjr) [inline src/views/Request.svelte]: looks like this could be a `const`

File: src/views/Request.svelte (hunk 1 above)
Comment (forevermatt) [reply src/views/Request.svelte]: I couldn't come up with a clear declarative way to do this, so for now I added a comment to at least provide a helpful mental clue as to what's happening. A better solution can be added later.

Comment (wcjr) [reply src/views/Request.svelte]: understood

**Pull request merged**